### PR TITLE
Feat/lesq 300/use new mvs

### PR DESCRIPTION
**Pull request by kimon-satan**
## Description

GraphQL files are updated to use the latest mvs from Database tools which work with the new published data.
Minor tweaks have been made to schemas to deal with omitted values and nullable fields
Some changes to the units page have been made to allow learningThemes to be nullable

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1910--oak-web-application.netlify.thenational.academy
2. Follow a user journey from keystage to lesson for the new content. Try combined science for a complex example. For an example with a semi-populated lesson try http://localhost:3000/teachers-2023/programmes/geography-secondary-ks4/units/test-unit/lessons/lesson-1-in-test-unit

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

File: src/components/ProgrammeList/ProgrammeListItem.tsx [hunk 1]
```diff
@@ -79,26 +73,7 @@ const ProgrammeListItem: FC<
...
-          $font={"body-3"}
-          $color={"oakGrey4"}
-        >
-          <Span $mb={4}>{`${unitCount} units`}</Span>
-          <Span $mh={4}>
-            <Icon name="dot" size={8} />
-          </Span>
-          <Span $font={"body-3"}>{`${lessonCount} lessons`}</Span>
-        </Flex>
-      </Box>
```
Comment (njblore) [inline src/components/ProgrammeList/ProgrammeListItem.tsx:101]: has this info moved somewhere else or are we just removing it?

Comment (kimon-satan) [reply src/components/ProgrammeList/ProgrammeListItem.tsx:101]: we're just removing it

**Review by njblore — APPROVED** [review]
Looks good! 👍

**Review by jack-skerrett-oak — APPROVED** [review]
Approving to unblock dev but there will need to be a couple of mop up tickets

**Pull request merged**